src/core/renderController.ts

```diff
@@ -89,23 +84,24 @@ module Plottable {
     export function flush() {
       if (_animationRequested) {
         // Layout
-        _componentsNeedingComputeLayout.values().forEach((component: Component) => component.computeLayout());
+        _componentsNeedingComputeLayout.values().forEach((component) => component.computeLayout());
```

Comment:
type the function. It's weird that `tsc` sometimes complains about this, sometimes it does not